wine-staging/PKGBUILD

```
@@ -0,0 +1,172 @@
+# Maintainer: Felix Yan <felixonmars@archlinux.org>
+# Maintainer: Peter Jung <ptr1337@archlinux.org>
+# Contributor: Sven-Hendrik Haase <sh@lutzhaase.com>
+# Contributor: Jan "heftig" Steffens <jan.steffens@gmail.com>
+# Contributor: Eduardo Romero <eduardo@archlinux.org>
+# Contributor: Giovanni Scafora <giovanni@archlinux.org>
+
+pkgname=wine-staging
+pkgver=10.3
+pkgrel=2
+
+_pkgbasever=${pkgver/rc/-rc}
+_winever=$_pkgbasever
+#_winever=${_pkgbasever%.*}
+
+source=("git+https://gitlab.winehq.org/wine/wine.git?signed#tag=wine-$_pkgbasever"
+        "git+https://gitlab.winehq.org/wine/wine-staging.git#tag=v$_pkgbasever"
+        30-win32-aliases.conf
+        wine-binfmt.conf)
+source+=(
+        wine-ntsync.patch
+)
```

I'm confused... why not just
```suggestion
source=("git+https://gitlab.winehq.org/wine/wine.git?signed#tag=wine-$_pkgbasever"
        "git+https://gitlab.winehq.org/wine/wine-staging.git#tag=v$_pkgbasever"
        30-win32-aliases.conf
        wine-binfmt.conf
        wine-ntsync.patch)
```

same with the rest of the arrays..

---

It helps with diffs as it creates less conflicts and more clear boundaries of the changes over the base. I also allows me to not change "unrelated" lines. The other reason is that these will likely be removed in a few versions when the patch is merged into mainline wine.

---

Okay then, it looked insane to me but that's a fair argument. These packages are also maintained by you so whatever is most comfortable for you is what's best :)